docs/reference/developer_guide/api.md

```diff
@@ -1,4 +1,4 @@
 # Shift API
 
 The developers documentation can be found at the following location in the installation folder:
-`docs/html/index.html`
+`<path_to_the_shift_installation_folder>/docs/html/index.html`

```

Comment:
```suggestion
`&ltpath_to_the_shift_installation_folder&gt/docs/html/index.html`
```

Reply:
This is showing up correctly
![image](https://github.com/user-attachments/assets/b0395f9f-09f6-42f9-bd52-74dc28df7d4e)